demos/background_subtraction_demo/cpp_gapi/include/custom_kernels.hpp

```diff
@@ -4,14 +4,21 @@
 
 #pragma once
 
-#include <opencv2/gapi/gkernel.hpp>
-#include <opencv2/gapi/infer/ie.hpp>
+#include <string>  // for string
+#include <vector>  // for vector
 
-#include <inference_engine.hpp>
+#include <cpp/ie_cnn_network.h>  // for CNNNetwork
+#include <ie_allocator.hpp>  // for InferenceEngine
+#include <ie_common.h>  // for OutputsDataMap
+#include <ie_input_info.hpp>  // for InputsDataMap
+#include <opencv2/core.hpp>  // for Size, CV_8U
+#include <opencv2/gapi/gkernel.hpp>  // for G_API_OP, KernelTypeMedium, GKernelPackage
+#include <opencv2/gapi/gmat.hpp>  // for GMatDesc, GMat
```

Comment:
We won't add comments explaining what the header is for in newly added files when writing new demos. This will result in some files having such comments and some not. May be remove these comments now?